test/functionalTest/cases/0787_cprs_full_functional/update_forward_value_in_ngsiv2.test

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test is showing a buggy behaviour, to be fixed in a separate issue. It should be removed from the PR (although keeping the file in a safe place, as it would be very useful for sure when dealing with that issue).

In addition, thinking in its future use, a couple of steps like 3-4 should be added, to check also for the text/plain case.